azure-simple/packer/variables.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
License header is missing.

Reply:
The generic license header throws an error with json files and needs to check any other way supporting to add the license.